charts/incubator/firezone/templates/_secrets.tpl

```diff
@@ -4,12 +4,12 @@
 
 # firezone requires all these keys to be in base 64 | b64enc format presented in the container, so this b64enc here is intentional
 # https://www.firezone.dev/docs/reference/env-vars#secrets-and-encryption
-{{- $keyGuardian := randAlphaNum 64 | b64enc -}}
-{{- $keyDatabase := randAlphaNum 64 | b64enc -}}
-{{- $keySecret := randAlphaNum 64 | b64enc -}}
-{{- $keyLive := randAlphaNum 64 | b64enc -}}
-{{- $keyCookieSigning := randAlphaNum 64 | b64enc -}}
-{{- $keyCookieEncrypt := randAlphaNum 64 | b64enc -}}
+{{- $keyGuardian := (printf "base64:%v" (randAlphaNum 32 | b64enc)) -}}
+{{- $keyDatabase := (printf "base64:%v" (randAlphaNum 32 | b64enc)) -}}
+{{- $keySecret := (printf "base64:%v" (randAlphaNum 32 | b64enc)) -}}
+{{- $keyLive := (printf "base64:%v" (randAlphaNum 32 | b64enc)) -}}
+{{- $keyCookieSigning := (printf "base64:%v" (randAlphaNum 32 | b64enc)) -}}
+{{- $keyCookieEncrypt := (printf "base64:%v" (randAlphaNum 32 | b64enc)) -}}

```

Comment:
the base64: is not needed :D
It was just some kind of "convention" in annonaddy

Reply:
wanted to try it atleast, firezone just never works.